miniserver/routes/miniID.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express')
+const { StatusCodes } = require('http-status-codes')
+// const { NFT } = require('../../server/src/data/nft')
+const router = express.Router()
+// const { hasUserSignedBody } = require('../../server/routes/middleware')
+// const sharedUtils = require('../../shared/utils')
+const { validateID } = require('./middleware')
+const blockchain = require('../blockchain')
+
+// allows an existing user to lookup another user's address by their phone number, iff the phone number exists and the target user does not choose to hide its phone-address mapping (under `hide` parameter in settings)
+router.post('/lookup', validateID, async (req, res) => {
+  console.log(`req.body: ${JSON.stringify(req.body)}`)
+  const { phone, address } = req.processedBody
+  return res.json({ phone: phone, address: address })
+})
+
+router.post('/getid', validateID, async (req, res) => {
+  // get MiniId contract
+  const { phone, address } = req.processedBody
+  const validMiniID = {}
```

Comment:
Generally, do not create an object then mutate its field, because it is hard to track and it loses all the semantic meaning for making it a const. The object is meant to be immutable. Create the in the end only when it is needed

Reply:
Good advice 🙏